### ISSUE TITLE
[Question] How can we prepare `table_monitors_config`?

### ISSUE THREAD

**Issue by yu-iskw** [elementary-data/elementary#58]
## Overview
I tried to monitor dbt tests with [jaffle_shopw](https://github.com/dbt-labs/jaffle_shop) by following [the documentation](https://docs.elementary-data.com/quickstart/install-and-configure). But, I was not able to upload artifacts due to the lack of the destination table. If I am correct, we have to create the table `table_monitors_config` a head. But, the documentation doesn't describes `table_monitors_config`. How can we prepare the table?

## Environments
- Python 3.8
- dbt 1.0.3
- elementary 0.3.2.

## Error message

```bash
09:22:03  Running 2 on-run-end hooks
09:22:33  1 of 2 START hook: jaffle_shop.on-run-end.0..................................... [RUN]
09:22:33  1 of 2 OK hook: jaffle_shop.on-run-end.0........................................ [OK in 0.00s]
09:22:33  2 of 2 START hook: elementary.on-run-end.0...................................... [RUN]
09:22:33  2 of 2 OK hook: elementary.on-run-end.0......................................... [OK in 0.00s]
09:22:33
09:22:33
09:22:33  Finished running 8 view models, 9 incremental models, 2 table models, 3 seeds, 17 tests, 3 hooks in 44.13s.
09:22:33
09:22:33  Completed with 2 errors and 0 warnings:
09:22:33
09:22:33  Runtime Error in model filtered_information_schema_columns (models/edr/metadata_store/filtered_information_schema_columns.sql)
09:22:33    404 Not found: Table sandbox-project:jaffle_shop_elementary.table_monitors_config was not found in location asia-northeast1
09:22:33
09:22:33    (job ID: d0f16aa6-b33d-493f-b0d1-8b934a090682)
09:22:33
09:22:33  Runtime Error in model filtered_information_schema_tables (models/edr/metadata_store/filtered_information_schema_tables.sql)
09:22:33    404 Not found: Table sandbox-project:jaffle_shop_elementary.table_monitors_config was not found in location asia-northeast1
09:22:33
09:22:33    (job ID: 03fc0b75-262f-49f8-8957-96b55268e9ac)
```

**Comment by yu-iskw** [comment]
@oravi That's true. I tried out it with BigQuery, not Snowfrake. 

I will leave another comment about the details of logs later. But, as far as I am concerned, `table_monitors_config ` is defined as a dbt source. I don't think it is created by `dbt run`. Am I correct?

https://github.com/elementary-data/dbt-data-reliability/blob/master/models/edr_sources/sources.yml#L7

**Comment by yu-iskw** [comment]
@oravi This is the full log of dbt.

https://github.com/yu-iskw/jaffle_shop/blob/048fa29b53fb105b516d5e6634b5325dc315000a/logs/dbt.log

**Comment by oravi** [comment]
@yu-iskw thanks for sharing it! the sources are being created in the on-run-start hook. They are supposed to be created in the target.database under '<target.schema>_elementary'. According to the log it looks like the on-run-start hook ran successfully. Is there a reason you can think of that it won't be created there or the others models won't have access to it?

In any case, we are doing final tests on BigQuery today and changing the flow of the configuration tables to be simpler to eliminate completely issues like that. So it will be resolved in the next version that should be released in a day or two.

**Comment by oravi** [comment]
Hi @yu-iskw ! happy to update that we have just released a new version that supports BigQuery and also do not rely on the configuration tables anymore. The configuration tables are now removed and the code basically uses the dbt graph to extract the tests configuration. 

Please note that we also updated the documentation so I'd recommend checking them out before upgrading, would love to hear your feedback and see if this resolves this issue.

Thanks!!

**Comment by yu-iskw** [comment]
@oravi Thank you for sharing. I will give a try the new version. I will let you know how it is later.

**Comment by Maayan-s** [comment]
Closing the issue, fixed on 0.3.8 and above :)
